src/mixins/enum-mixin.ts

```diff
@@ -155,6 +155,11 @@ export default class EnumMixin extends Vue {
     return (item.name === FilingTypes.PUT_BACK_ON)
   }
 
+  /** Return True if the filing is a Admin Freeze */
+  isTypeAdminFreeze (item: any): boolean {
+    return (item.name === FilingTypes.ADMIN_FREEZE)
+  }
+
```

Comment:
You should update the method on line 236 (and its unit test).

Also, is any special needed to display this filing in the Recent Filing History?

Reply:
![image](https://user-images.githubusercontent.com/295819/212209828-6b5c1fe6-1bcc-4caf-8e5a-c85fac256230.png)

Reply:
this will appear like this ^ above screenshot

Reply:
Thanks! A picture is worth 1000 words. This shows the new alert and the ledger item in the history 👍 

However, I notice here that the Business Address Change and Proprietor Change buttons are still enabled, but they should be disabled.

Reply:
Did updating the "blocker" getter fix the Business Address Change and Proprietor Change buttons?

Reply:
Yes it did Severin:

![image](https://user-images.githubusercontent.com/3484109/213254738-5188f6f1-fd10-4ed5-9a06-6e9a08f247d5.png)